tracing-jersey/src/main/java/com/palantir/tracing/jersey/TraceEnrichingFilter.java

```diff
@@ -128,7 +129,7 @@ private static Observability getObservabilityFromHeader(ContainerRequestContext
 
         @Override
         public <T> void translate(TagAdapter<T> adapter, T target, ContainerResponseContext data) {
-            adapter.tag(target, "status", Integer.toString(data.getStatus()));
+            adapter.tag(target, TraceTags.HTTP_STATUS_CODE, Integer.toString(data.getStatus()));
```

Comment:
Now that makes me think we should've made TraceTag an interface, so that TraceTags is discoverable...

Reply:
Perhaps, although I expect most folks won't use any of these tags because they have more specific information. I think these will primarily be helpful for us :-)